EmpireLegends/PlayerStrategies.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+#include <map>
```

Comment:
Missing comments for most of the things

Reply:
Methods are all inherited, want me to repeat the comments?

Reply:
uhh oh good point. im not sure. guess not.

Reply:
I'll add comments on the non human classes, but not on the methods